localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
+ jarldom_frost_vrykul_duchy:0 "Jarldom"
+ high_king_male_frost_vrykul_emperor:0 "High King"
+ high_queen_female_frost_vrykul_emperor:0 "High Queen"
+ high_kingdom_frost_vrykul_empire:0 "High Kingdom"
+
+ ### Staroveric ###
+ # Feudal
+ grand_prince_male_solstice_duke:0 "Grand Prince"
+ grand_princess_female_solstice_duke:0 "Grand Princess"
+ grand_principality_solstice_duchy:0 "Grand Principality"
+ tsar_male_solstice_emperor:0 "Tsar"
+ tsaritsa_female_solstice_emperor:0 "Tsaritsa"
+ tsardom_solstice_empire:0 "Tsardom"
+
+ ### Dryad group ###
+ # Feudal / Republic / Clan / Theocratic / Tribal
+ tender_male_dryad_group_baron:0 "Tender"
+ tender_female_dryad_group_baron:0 "Tender"
+ refuge_dryad_group_baronny:0 "Refuge"
+ guardian_male_dryad_group_count:0 "Guardian"
+ guardian_female_dryad_group_count:0 "Guardian"
+ grove_dryad_group_county:0 "Grove"
+ lord_male_dryad_group_duke:0 "Lord"
+ lady_female_dryad_group_duke:0 "Lady"
+ grand_grove_dryad_group_duchy:0 "Grand Grove"
+ high_lord_male_dryad_group_king:0 "High Lord"
+ high_lady_female_dryad_group_king:0 "High Lady"
+ domain_dryad_group_kingdom:0 "Domain"
+ king_male_dryad_group_emperor:0 "King"
+ queen_female_dryad_group_emperor:0 "Queen"
+ kingdom_dryad_group_empire:0 "Kingdom"
+
+ ### Tol'vir ###
+ # Feudal - Normal
+ hatya_male_tolvir_baron:0 "Hatya"
+ hatya_female_tolvir_baron:0 "Hatya"
+ niout_tolvir_barony:0 "Niout"
+ heqa_male_tolvir_count:0 "Heqa"
+ heqa_female_tolvir_count:0 "Heqa"
+ hout_tolvir_county:0 "Hout"
+ nomarch_male_tolvir_duke:0 "Nomarch"
+ nomarch_female_tolvir_duke:0 "Nomarch"
+ nome_tolvir_duchy:0 "Nome"
+ pharaoh_male_tolvir_king:0 "Pharaoh"
+ pharaoh_female_tolvir_king:0 "Pharaoh"
+
+ # Feudal - Evil religions
+ dark_pharaoh_male_tolvir_king:0 "Dark Pharaoh"
+ dark_pharaoh_female_tolvir_king:0 "Dark Pharaoh"
+
+ vizier_male_tolvir:0 "Vizier"
+ vizier_female_tolvir:0 "Vizier"
+ grand_vizier_male_tolvir:0 "Grand Vizier"
+ grand_vizier_female_tolvir:0 "Grand Vizier"
+
+ ### Mogus and Pandarens ###
+ # Feudal
+ zhang_male_mogu_baron:0 "Zhang"
+ zhang_female_mogu_baron:0 "Zhang"
+ district_mogu_baronny:0 "District"
+ taishou_male_mogu_count:0 "Taishou"
+ taishou_female_mogu_count:0 "Taishou"
+ commandery_mogu_county:0 "Commandery"
+
+ # Pandaren specific
+ mu_male_pandaren_duke:0 "Mu"
+ mu_female_pandaren_duke:0 "Mu"
+ province_pandaren_duchy:0 "Province"
+
+ # Mogu specific
+ warlord_male_mogu_duke:0 "Warlord"
+ warlady_female_mogu_duke:0 "Warlady"
+ clan_mogu_duchy:0 "Clan"
+
+ ### Jinyu ###
+ # Feudal / Clan
+ joshu_male_jinyu_baron:0 "Joshu"
+ joshu_female_jinyu_baron:0 "Joshu"
+ shiro_jinyu_barony:0 "Shiro"
+ kokujin_male_jinyu_count:0 "Kokujin"
+ kokujin_female_jinyu_count:0 "Kokujin"
+ kori_jinyu_county:0 "Kori"
+ daimyo_male_jinyu_duke:0 "Daimyo"
+ daimyo_female_jinyu_duke:0 "Daimyo"
+ clan_jinyu_duchy:0 "Clan"
+ shogun_male_jinyu_king:0 "Shogun"
+ shogun_female_jinyu_king:0 "Shogun"
+ shogunate_jinyu_kingdom:0 "Shogunate"
```

Comment:
I want again remind you of the thing i wrote on discord, becouse i think it fits well; 

> I'm just asking if y'all know the difference between a shogunate and a kingdom? because a shogunate is like a Grand Marshal or regent(Sessho?) ruling the nation while there is a more legitimate ruler that have no power or their power ain't as legitimate as a monarch. The word shogun is from 将 Sho (commander/general) and 軍 Gun (troops, army, military) so it is indented to be a very militaristic title . 
> the Japanese word for a normal king is 王 = Ou = ō or 王様 (ō-sama). This is not a uncommon term in japan, it's what they would use as a term for "kings" outside of japan. Kingdom is 王国 (okoku). Some nations that did no acknowledge the japanese emperor as an emperor called them 日本国王 (Nihon Kokuo, King of Japean) (国 koku, country 王 Ou/ō, king). As someone who knows a little Japanese I think having a more monarch title for the kingship works & sounds better, because a shogun is a title from a far more complex government form then a normal kingdom. I would suggest to have Ou/ō or Kokuo/ō as the king tiles, this also give us princess/prince titles such as prince =王子 oji. but i guess shogun works in the rule of cool astatic

https://jitenon.com/word/6677
https://jitenon.com/word/13498
https://jitenon.com/word/2941
https://www.japanese-wiki-corpus.org/history/King%20of%20Japan%20(Nihon%20Kokuo).html
https://www.japanese-wiki-corpus.org/title.html

Reply:
Well, done in 78ec9b158